sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/CertificateProperties.java

```diff
@@ -35,17 +35,17 @@
     /**
      * Expiry date in UTC.
      */
-    private OffsetDateTime expires;
+    private OffsetDateTime expiresOn;
```

Comment:
can you confirm that we landed on "On" and not "At"

Reply:
KV is using "On" suffix across all languages.
Storage and Identity too use "On" suffix, I believe.